src/App.tsx

```diff
@@ -1,22 +1,72 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { TodoFooter } from './components/TodoFooter';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { TodoErrors } from './components/TodoErrors';
+import { Status } from './types/Status';
 
-const USER_ID = 0;
+const USER_ID = 11226;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [status, setStatus] = useState<Status>(Status.all);
+
+  const filteredTodos = todos.filter(todo => {
+    switch (status) {
+      case Status.completed:
+        return todo.completed;
+
+      case Status.active:
+        return !todo.completed;
+
+      default:
+        return true;
+    }
+  });
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => {
+        setError('Error');
```

Comment:
The error message is too generic. Consider providing more specific error messages for better debugging and user experience.